### ISSUE TITLE
Não funciona mais

### ISSUE THREAD

**Issue by lusbite** [leodlabs/kuduairo-translator-api#1]
Já tem umas semanas que tento usar o tradutor, e ele simplesmente não funciona. Ele fica eternamente na tela "Fazoaindo cieincia..."
![image](https://github.com/leodlabs/kuduairo-translator-api/assets/42288040/81eaf6e1-036e-49d2-ae57-d90df2d509f9)

**Comment by klausmuller66** [comment]
pois é, pra mim também. isso já chegou a funcionar?

**Comment by leodhb** [comment]
Sim, já chegou a funcionar. A [API de tradução](https://github.com/leodlabs/kuduairo-translator-api) era hospedada no heroku, e desde que eles acabaram com o plano grátis a API deixou de estar no ar. Vou tentar subir em alguma outra plataforma como o Render e ver se dá bom

**Comment by sz-ricardo-cardozo** [comment]
Poe na aws
